specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -873,6 +873,16 @@
               "type": "boolean"
             }
           }
+        },
+        "dotNet": {
+          "description": ".NET app configuration",
+          "type": "object",
+          "properties": {
+            "autoConfigureDataProtection": {
+              "description": "Auto configure the ASP.NET Core Data Protection feature",
```

Comment:
>feature [](http://example.com/codeflow?start=78&length=7)

Do you want to document a default value/behavior for this api version, or is unspecified more like 'unspecified, let the platform choose'?

Reply:
Thanks Tim! Yes, I initially declared a `default` value but it created other issues (cf. https://github.com/Azure/azure-rest-api-specs/pull/28001#discussion_r1508624160).